### PR TITLE
feat: basic input component added

### LINKED ISSUE #25: Add Input Component
image
image
image

### PR DESCRIPTION
**Pull request by AlejandroSalcido**
Hello Team, i have built the basic structure of the input component, i made it responsive to every different phone where it is displayed, in order to offer a better UX, please let me know what you think about this implementation, in order for me to proceed with the other input functions!

### CODE REVIEW

File: src/components/Input.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components'
+
+const Input = () => {
+  return (
+    <div>
```
Comment (Israel-Laguan) [inline src/components/Input.jsx]: this div is not necessary as Wrapper is a div already

File: src/components/Input.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components'
+
+const Input = () => {
```
Comment (Israel-Laguan) [inline src/components/Input.jsx]: Please add appropriate props like "type", "placeholder", "value", "setValue" (to map to `onChange`), "id", "name", etc.

File: src/components/Input.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <div>
+      <InputWrapper>
+        <InputData></InputData>
```
Comment (Israel-Laguan) [inline src/components/Input.jsx]: input usually doesn't have children, please check if closing tag is needed

**Comment by Israel-Laguan** [comment]
Please rebase

**Comment by Israel-Laguan** [comment]
Please notice @ginnaaceror included some Texts using input styles, what do you think about it, @AlejandroSalcido ?

https://github.com/Azordev/dasher-user/blob/a13d85e964842b551c24734027ba673eef2c1cc1/src/pages/Confirm.jsx#L21-L31

File: src/components/Input.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+const Input = ({ name, type, placeholder, onChange, value, setValue }) => {
+  return (
+    <InputWrapper>
+      <InputData
+        id={name}
+        name={name}
+        type={type}
+        placeholder={placeholder}
+        onChnage={onChange}
```
Comment (Israel-Laguan) [inline src/components/Input.jsx]: typo here

File: src/components/Input.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        value={value}
+        setValue={setValue}
```
Comment (Israel-Laguan) [inline src/components/Input.jsx]: this is not a valid artribute

File: src/components/Input.jsx (hunk 4 above)
Comment (Israel-Laguan) [reply src/components/Input.jsx]: please try

onChange={setValue} 

or similar

File: src/components/Input.jsx (hunk 5 above)
Comment (AlejandroSalcido) [reply src/components/Input.jsx]: i deleted it just left it on props

File: src/components/Input.jsx (hunk 4 above)
Comment (AlejandroSalcido) [reply src/components/Input.jsx]: Checked, Thanks for the advice!

**Review by ginnaaceror — COMMENTED** [review]
Hi, I can't see the last changes that are in the dev branch.
@AlejandroSalcido what is the path to see the input?

**Comment by AlejandroSalcido** [comment]
> Hi, I can't see the last changes that are in the dev branch. @AlejandroSalcido what is the path to see the input?

I think that is because it isn't megred yet Ginna

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**